### PR TITLE
BACKLOG-20484 Update label, moonstone

### PR DESCRIPTION
**Pull request by AKarmanov**
## JIRA

https://jira.jahia.org/browse/BACKLOG-20484

## Description

WARNING!!! Do not merge until moonstone is updated

### CODE REVIEW

File: src/javascript/SelectorTypes/MultipleLeftRightSelector/MultipleLeftRightSelector.jsx
Comment (tdraier) [inline src/javascript/SelectorTypes/MultipleLeftRightSelector/MultipleLeftRightSelector.jsx:32]: That test is probably not needed, _plural is already added by i18next based on count :
```suggestion
    const selectedLabel = t(`${labelBase}.selected`, {count: arrayValue.length});
```

**Comment by gflores-jahia** [comment]
Waiting on moonstone release before merging https://github.com/Jahia/moonstone/pull/343

**Review by tdraier — APPROVED** [review]

**Pull request merged**
